src/server/config-dist.json

```diff
@@ -15,5 +15,83 @@
 		"LED_BRIGHTNESS": 255,
 		"LED_INVERT": false,
 		"LED_CHANNEL": 0
+	},
+	"SENSORS": {
+		"0x48": {
+			"HARDWARE_NAME": "ADS1115",
+			"CONNECTED_CHANNELS": [0,1,2],
+			"GAINS": [1,1,1],
+			"R1_VALUES": [22,22,22],
+			"R2_VALUES": [3.3,3.3,3.3],
+			"CORRECTION_FACTORS": [1.0065,1.0127175 , 1.02739],
+			"LABELS": ["Supply [v]","Battery 1 [v]","Battery 2 [v]"],
+			"ADDRESS": 72
+		},
+		"0x49": {
+			"HARDWARE_NAME": "ADS1115",
+			"CONNECTED_CHANNELS": [0,1,2],
+			"GAINS": [1,1,1],
+			"R1_VALUES": [22,22,22],
+			"R2_VALUES": [3.3,3.3,3.3],
+			"CORRECTION_FACTORS": [1.0065,1.0127175 , 1.02739],
+			"LABELS": ["Supply [v]","Battery 1 [v]","Battery 2 [v]"],
+			"ADDRESS": 73
```

Comment:
Can we not get the address from the key? python should have a parse hex string function.